src/_data/sidenav.yml

```diff
@@ -145,6 +145,8 @@
     - title: Add Flutter to existing app
       permalink: /docs/development/add-to-app
       children:
+      - title: Introduction
+        permalink: /docs/development/add-to-app/index
```

Comment:
You shouldn't have to specify index. Just /docs/development/add-to-app should be sufficient.

Reply:
ah sg, thanks!